backend/game/turn_actions/attack.py

```diff
@@ -1,5 +1,10 @@
+from game.values.locations import Locations
+from game.turn_actions.player_movement import move_to_tokyo_if_empty, yield_tokyo
+
+
 def is_attackable(attacking_player_location, other_player):
-    if attacking_player_location != other_player.location:
+    if attacking_player_location != other_player.location and \
+            (attacking_player_location == Locations.OUTSIDE or other_player.location == Locations.OUTSIDE):
```

Comment:
I question whether this condition needs to be this complicated, but regardless this needs to be refactored into a method to make the code readable.

Reply:
I added the extra complexity for when Tokyo bay eventually gets implemented. My thought was this could cause a sneaky bug to track down later where monsters in Tokyo and Tokyo Bay would attack each other so I thought I would handle it now.

Reply:
I think the new version is more legible